deep_qa/models/multiple_choice_qa/tuple_inference.py

```diff
@@ -66,27 +69,36 @@ def _instance_type(self):
     @classmethod
     @overrides
     def _get_custom_objects(cls):
-        custom_objects = super(MultipleChoiceTupleInferenceModel, cls)._get_custom_objects()
+        custom_objects = super(TupleInferenceModel, cls)._get_custom_objects()
         custom_objects['TupleMatch'] = TupleMatch
+        custom_objects['MaskedSoftmax'] = MaskedSoftmax
+        custom_objects['NoisyOr'] = NoisyOr
+        custom_objects['Repeat'] = Repeat
+        custom_objects['Squeeze'] = Squeeze
         return custom_objects
 
     @overrides
     def _get_max_lengths(self) -> Dict[str, int]:
-        return {
-                'num_question_tuples': self.num_question_tuples,
-                'num_background_tuples': self.num_background_tuples,
-                'num_slots': self.num_tuple_slots,
-                'word_sequence_length': self.max_slot_length,
-                'num_options': self.num_options,
-                }
+        max_lengths = super(TupleInferenceModel, self)._get_max_lengths()
+        max_lengths['num_question_tuples'] = self.num_question_tuples
+        max_lengths['num_background_tuples'] = self.num_background_tuples
+        max_lengths['num_slots'] = self.num_tuple_slots
+        max_lengths['word_sequence_length'] = self.num_slot_words
+        max_lengths['num_options'] = self.num_options
+        if self.word_character_length is not None:
```

Comment:
This happens in the super class.  You only need to worry about the things you define in `self.__init__`.